packages/terra-application-navigation/src/extensions/_TempSVG.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```

Comment:
Are we waiting for a name for this file or going with `_TempSVG.jsx`?

Reply:
Originally we were unsure if UX was going to add the icon or keep it private, so it was temporary. I'll update the name to something more manageable until the decision is made.

Reply:
Updated.